assets/js/blocks/order-confirmation/status/block.json

```diff
@@ -7,7 +7,38 @@
 	"keywords": [ "WooCommerce" ],
 	"supports": {
 		"multiple": false,
-		"align": [ "wide", "full" ]
+		"align": [ "wide", "full" ],
+		"html": false,
+		"typography": {
+			"fontSize": true,
+			"lineHeight": true,
+			"__experimentalFontFamily": true,
+			"__experimentalTextDecoration": true,
+			"__experimentalFontStyle": true,
+			"__experimentalFontWeight": true,
```

Comment:
I cannot see this option on the site. I tested the PR using the following themes:
- [Twenty Twenty-Three](https://wordpress.org/themes/twentytwentythree/)
- [Twenty Twenty-Two](https://wordpress.org/themes/twentytwentytwo/)
- [Abisko](https://wordpress.org/themes/abisko/)
- [Beaumont](https://wordpress.org/themes/beaumont/)

Reply:
It appears under _Appearance_ on the typography section and adds additional options such as **bold**.